src/components/HierarchyPicker/HierarchyPicker.js

```diff
@@ -12,26 +12,31 @@ export default function HierarchyPicker({
   const hierarchies = [
     {
       name: 'Leaf',
+      displayName: 'Leaf',
       icon: 'leaf.svg',
       description: 'small goal',
     },
     {
       name: 'Branch',
+      displayName: 'Branch',
       icon: 'branch-with-leaf.png',
       description: 'sub-goal',
     },
     {
       name: 'Trunk',
+      displayName: 'Trunk',
       icon: 'trunk.png',
       description: 'high-level goal',
     },
     {
       name: 'Root',
+      displayName: 'Root',
       icon: 'root.png',
       description: 'primary goal',
     },
     {
-      name: 'No Hierarchy',
+      name: 'NoHierarchy',
```

Comment:
`name` is what it's like in the backend
`displayName` is how it should appear on screen